### PR TITLE
feat(frontends): Implemented acos for Paddle.tensor.tensor.Tensor 

### LINKED ISSUE #25916: acos

### PR DESCRIPTION
**Pull request by Adeola23**
# PR Description

Added acos to paddle.tensor.tensor.Tensor class

## Related Issue

Close #25916

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

-->

### CODE REVIEW

**Comment by Adeola23** [comment]
@bipinKrishnan please review

**Comment by Adeola23** [comment]
@bipinKrishnan

**Comment by bipinKrishnan** [comment]
ivy-gardener

**Comment by Adeola23** [comment]
ivy-gardener

**Comment by Adeola23** [comment]
@bipinKrishnan

**Comment by Adeola23** [comment]
@bipinKrishnan

**Comment by Adeola23** [comment]
@bipinKrishnan

**Comment by Adeola23** [comment]
@bipinKrishnan pls review

**Comment by Adeola23** [comment]
@bipinKrishnan pls review

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -751,7 +751,11 @@ def nonzero(self):
 
     @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
     def inner(self, y, name=None):
-        return paddle_frontend.inner(self, y, name)
+        return paddle_frontend.inner(self._ivy_array, y, name)
+
+    @with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
```
Comment (bipinKrishnan) [inline ivy/functional/frontends/paddle/tensor/tensor.py:756]: Looking at the [paddle docs](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/acos_en.html#acos), seems like its better to use `with_supported_dtypes` decorator with float32, float64 and float16.

Comment (Adeola23) [reply ivy/functional/frontends/paddle/tensor/tensor.py]: @bipinKrishnan pls review my latest PR

**Comment by Adeola23** [comment]
@bipinKrishnan

**Comment by Adeola23** [comment]
@bipinKrishnan

**Comment by Adeola23** [comment]
@bipinKrishnan Please review

**Comment by Adeola23** [comment]
@bipinKrishnan could you pls review, thanks.

**Comment by Adeola23** [comment]
@bipinKrishnan pls review

**Comment by Adeola23** [comment]
@bipinKrishnan  pls review.

**Comment by Adeola23** [comment]
@bipinKrishnan it's been 2 weeks now without a response.

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 2]
```diff
@@ -766,7 +766,11 @@ def nonzero(self):
 
     @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
     def inner(self, y, name=None):
-        return paddle_frontend.inner(self, y, name)
+        return paddle_frontend.inner(self._ivy_array, y, name)
```
Comment (bipinKrishnan) [inline ivy/functional/frontends/paddle/tensor/tensor.py:769]: Seems like these are not related to your changes, you can undo these changes.

**Comment by Adeola23** [comment]
@bipinKrishnan
